### ISSUE TITLE
Center "Add new" button under text on the recurring_expenses index view

### ISSUE THREAD

**Issue by iokiwi** [iokiwi/moneyapp#30]
If you go to the recurring expenses screen there are no buttons or links to do anything useful.

`https://localhost:8000/recurring_expenses/`

We should add an "Add New" button to create a new recurring expense 

The functionality already exists and is fully operational...there's just no button to get there.

The url for the new expense view should be `/recurring_expenses/new` but you can refer to `app/recurring_expenses/urls.py` to double check.

**Comment by ankitgadling** [comment]
hello @iokiwi I would like the contribute the the project can you please assign this issue to me

**Comment by iokiwi** [comment]
Feel free to @ me if you have any issues getting setup. It should hopefully be as simple as doing a `docker-compose up` as per the readme.

**Comment by ankitgadling** [comment]
hey @iokiwi I am getting these logs in docker 
![error in docker](https://github.com/iokiwi/moneyapp/assets/86833325/2e4e98a9-e1fc-4d5b-b8bd-8a443addd840)

**Comment by iokiwi** [comment]
I will need some more info. I have tested this for macOS and WSL/Ubuntu and it should 'just work'.

Can you tell me more about your development environment? I can see you are using docker desktop so I you are on either windows or mac.

 * Can you confirm what operating system are you using? macOS? Windows with WSL? What distro?
 * Can you confirm that you built the image by running 'docker-compose up' from the root of the repository?
 * What was the output from that?

You might try deleting your image and trying again.

This command will delete the image

```bash
docker rmi -f $(docker image ls | grep moneyapp-app | awk '{ print $3 }')
```

Then you can try rebuild it by running this command from the root of the repository.

```bash
docker-compose build app
```

If you still have the error I need to see the output of the above command

**Comment by ankitgadling** [comment]
Hello @iokiwi  

- I am using Windows with WSL version 2
- Yes i have build the image using 'docker-compose up' from the root of the repository
- I have tried the above steps that you have mentioned in the comment but still the app is not running and I got the below output 
![error in docker 2](https://github.com/iokiwi/moneyapp/assets/86833325/20668dfd-d1b0-45fe-89fe-cb7cdb67df73)

**Comment by iokiwi** [comment]
It looks to me like you are using 'Git Bash'. I can't see your task bar or title bar but the purple 'MINGw64' gives it away for me. I imagine you can see the following icon in your taskbar and title bar?

**Can you confirm?**
![image](https://github.com/iokiwi/moneyapp/assets/1480457/53c6ec2d-7d45-4c2a-9dcb-109cc6a78672)

By comparison, WSL should look something like this

![image](https://github.com/iokiwi/moneyapp/assets/1480457/118cbd54-2467-42d3-b50b-c57f9cf1bca2)


Can you also please show me the output of these commands run from the root of the repository. These are to check that the entrypoint file and the app directory exist in the image. 

```bash
docker-compose run --entrypoint="/bin/ls" app /
```

```bash
docker-compose run --entrypoint="/bin/ls" app
```

## Recommendation:

Make sure you are actually using WSL.  It appears to me that you are not using WSL which is most likely the cause for your problems.

This article from Microsoft explains how to get WSL if you don't have it https://learn.microsoft.com/en-us/windows/wsl/install

If you have WSL then you can list available distributions and start one from powershell

In powershell, try just running

```
PS C:\Users\smerrick> wsl
```

This will launch the default WSL distribution - whatever that is for you.

You can list available WSL distributions and see what the default is with this command

```pwsh
PS C:\Users\smerrick> wsl --list
Windows Subsystem for Linux Distributions:
Ubuntu-22.04 (Default)
docker-desktop-data
docker-desktop
```

You can install a WSL distribution with a command like this.

```
PS C:\Users\smerrick> wsl --install Ubuntu-22.04
```


You can launch a specific distribution bu name with the -d `flag`

```
PS C:\Users\smerrick> wsl -d Ubuntu-22.04
```

## TL;DR please ensure you are using WSL and try again

**Comment by ankitgadling** [comment]
Hello @iokiwi I have successfully build the project on my  machine I am seeing that add new button is already implemented
![solved feature](https://github.com/iokiwi/moneyapp/assets/86833325/ffbdaeca-85cc-4466-96ed-0999b4b43fd8)

**Comment by iokiwi** [comment]
congratulations on getting up and running. you had some challenges but we got there!

Sorry the button must have snuck in with another pull request in the meantime! Lets find some more things you can work on!

**Comment by iokiwi** [comment]
if you did still want to make a contribution relating to the button, I would love to see the button horizontally centered on the page underneath the text.

**Comment by ankitgadling** [comment]
hello @iokiwi I have centered a button 
![center add new button](https://github.com/iokiwi/moneyapp/assets/86833325/57c16abc-5e80-4baa-a93a-78a520514ab6)

**Comment by iokiwi** [comment]
Amazing! Great work it looks fantastic :)

Now you will want to raise a pull request to merge your branch on your fork into the main branch in my repository

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request
